Sources/Client.swift

```diff
@@ -91,17 +92,16 @@ open class Client {
         }
     }
 
-    fileprivate func URLForFragment(fragment: String = "", parameters: [String: Any]? = nil) -> URL? {
-        if var components = URLComponents(string: "\(scheme)://\(server)/spaces/\(spaceIdentifier)/\(fragment)") {
+    fileprivate func URL(forComponent component: String = "", parameters: [String: Any]? = nil) -> URL? {
```

Comment:
This does not change is only internal, aka no user will use `URLForFragment` in their code, right?